packages/design-system/src/components/WIP/Accordion/Primitive/CollapsiblePanelHeader.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { cloneElement, forwardRef, MouseEvent, ReactElement, Ref } from 'react';
+import { Clickable } from 'reakit/Clickable';
+import classnames from 'classnames';
+import { ButtonIcon } from '../../../ButtonIcon';
+import { SizedIcon } from '../../../Icon';
+import tokens from '@talend/design-tokens';
+
+import styles from './CollapsiblePanelHeader.module.scss';
+
+type CollapsiblePanelHeaderPropsType = {
+	size?: 'S' | 'M';
+	expanded: boolean;
+	title: string;
+	metadata?: ReactElement[];
+	action?: {
+		icon: string;
+		tooltip: string;
+		callback: () => unknown;
+	};
```

Comment:
If we extract that we can reuse it for the parent `action` prop then instead of the any or my suggestions